test/acceptance/features/steps/knative_serving.py

```diff
@@ -0,0 +1,39 @@
+import re
+from command import Command
+from openshift import Openshift
+from pyshould import should_not
+
+
+class Knative_Serving():
```

Comment:
Can we name the class as `KnativeServing`, so that we follow the same pattern as with other classes introduced in this PR?

Reply:
Sure